Fix:

bdump.c:183:22: error: implicitly declaring library function 'isdigit' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]

bdump.c:190:17: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

bdump.c:244:50: error: implicitly declaring library function 'tolower' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]

bdump.c:448:25: error: implicitly declaring library function 'isprint' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]

--- bdump.c.orig	2004-12-22 01:37:58.000000000 -0600
+++ bdump.c	2021-02-14 01:18:19.000000000 -0600
@@ -14,8 +14,10 @@
 /*==============================================================================
 Include Files
 ==============================================================================*/
+#include <ctype.h>
 #include <stdio.h>
 #include <stdarg.h>
+#include <stdlib.h>
 #include <string.h>
 #include <errno.h>
 #include <unistd.h>
